gateway-ha/src/main/java/io/trino/gateway/ha/clustermonitor/ServerInfo.java

```diff
@@ -13,27 +13,11 @@
  */
 package io.trino.gateway.ha.clustermonitor;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-import static java.util.Objects.requireNonNull;
-
 // based on https://github.com/trinodb/trino/blob/439/client/trino-client/src/main/java/io/trino/client/ServerInfo.java
 // without unused fields
-public class ServerInfo
+public record ServerInfo(
+        @JsonProperty("starting") boolean starting)
```

Comment:
Do we still need `@JsonProperty`?

Reply:
I thought that is the coding style for Trino to write all the `@JsonProperty` explicitly.

Reply:
We usually don't use `@JsonProperty` for record in Trino.